hicommonwealth/commonwealth
# Removed `/whyCommonwealth` page

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8257

## Description of Changes
Removed `/whyCommonwealth` page, which is now handled in the common.xyz domain

## "How We Fixed It"
N/A

## Test Plan
- Visit `/whyCommonwealth`
- Verify you see page not found message

## Deployment Plan
N/A

## Other Considerations
N/A
